src/Raven.Client/Documents/Operations/Backups/BackupSettings.cs

```diff
@@ -6,6 +6,8 @@ public abstract class BackupSettings
     {
         public bool Disabled { get; set; }
 
+        public string Name { get; set; }
```

Comment:
Probably better to have this as a `abstract` property with just `get`

Reply:
This isn't needed in the client (the name for the server only)